-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-41697/35569

ERROR: Build errors found during clang-tidy run.
```
--
L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc:526:19: error: no member named 'ecal_phi' in 'l1tpf_calo::CombinedCluster' [clang-diagnostic-error]
          cluster.ecal_phi = cluster.phi;
          ~~~~~~~ ^
L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc:555:19: error: no member named 'ecal_eta' in 'l1tpf_calo::CombinedCluster'; did you mean 'ecal_et'? [clang-diagnostic-error]
          cluster.ecal_eta = cluster.eta;
                  ^~~~~~~~
                  ecal_et
--
L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc:556:19: error: no member named 'ecal_phi' in 'l1tpf_calo::CombinedCluster' [clang-diagnostic-error]
          cluster.ecal_phi = cluster.phi;
          ~~~~~~~ ^
L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc:616:9: error: no member named 'ecal_eta' in 'l1tpf_calo::CombinedCluster'; did you mean 'ecal_et'? [clang-diagnostic-error]
    dst.ecal_eta = src.eta;
        ^~~~~~~~
        ecal_et
--
L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc:617:9: error: no member named 'ecal_phi' in 'l1tpf_calo::CombinedCluster' [clang-diagnostic-error]
    dst.ecal_phi = src.phi;
    ~~~ ^
Suppressed 552 warnings (552 in non-user code).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc
+++ b/L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc
@@ -589,6 +589,8 @@ void l1tpf_calo::SimpleCaloLinker::run() {
           float wecal = cluster.ecal_et / cluster.et, whcal = 1.0 - wecal;
           cluster.eta = ecal.eta * wecal + hcal.eta * whcal;
           cluster.phi = ecal.phi * wecal + hcal.phi * whcal;
+          cluster.ecal_eta = cluster.eta;
+          cluster.ecal_phi = cluster.phi;
           // wrap around phi
           cluster.phi = reco::reduceRange(cluster.phi);
           cluster.constituents.emplace_back(-i - 1, 1);
@@ -617,6 +619,8 @@ void l1tpf_calo::SimpleCaloLinker::run() {
           cluster.et = myet + etot;
           cluster.eta = hcal.eta + avg_eta / cluster.et;
           cluster.phi = hcal.phi + avg_phi / cluster.et;
+          cluster.ecal_eta = cluster.eta;
+          cluster.ecal_phi = cluster.phi;
           // wrap around phi
           cluster.phi = reco::reduceRange(cluster.phi);
         }
@@ -676,6 +680,8 @@ void l1tpf_calo::FlatCaloLinker::run() {
     dst.et = src.et;
     dst.eta = src.eta;
     dst.phi = src.phi;
+    dst.ecal_eta = src.eta;
+    dst.ecal_phi = src.phi;
     dst.ecal_et = 0;
     dst.hcal_et = 0;
     for (const auto &pair : src.constituents) {

